getting-started/dev/java.md

```diff
@@ -42,20 +36,26 @@ Suppose we wish to run inference on a document using Lucidtech’s invoice model
 
```

Comment:
Remove 'invoice' reference here. 'using one of the available models'. Maybe add a small example on how to use listModels() to get modelIds?